plugin/documents.py

```diff
@@ -169,7 +162,6 @@ def _cleanup(self) -> None:
         triggers = [trigger for trigger in triggers if 'server' not in trigger]
         settings.set("auto_complete_triggers", triggers)
         self._stored_region = sublime.Region(-1, -1)
-        self._color_phantoms.update([])
```

Comment:
where did this line go?

Reply:
I messed this up during a rebase when I was splitting up the PR, sorry. I've reverted it.